extensions/vertx/deployment/src/test/java/io/quarkus/vertx/deployment/MessageConsumerFailureTest.java

```diff
@@ -35,12 +38,15 @@
 
     @Test
     public void testFailure() throws InterruptedException {
-        verifyFailure("foo", "java.lang.IllegalStateException: Foo is dead");
-        verifyFailure("foo-message", "java.lang.NullPointerException");
-        verifyFailure("foo-completion-stage", "java.lang.NullPointerException: Something is null");
+        verifyFailure("foo", "java.lang.IllegalStateException: Foo is dead", false);
+        verifyFailure("foo-message", "java.lang.NullPointerException", false);
+        verifyFailure("foo-completion-stage", "java.lang.NullPointerException: Something is null", false);
+        verifyFailure("foo-completion-stage-failure", "boom", true);
+        verifyFailure("foo-uni", "java.lang.NullPointerException: Something is null", false);
```

Comment:
Instead of booleans, it would be better to pass the expected `replyException.failureCode()`. That would get rid of the `if`statement in the `verifyFailure` method